package.json

```diff
@@ -17,27 +17,27 @@
     "url": "https://github.com/typescript-eslint/typescript-eslint/issues"
   },
   "scripts": {
-    "build": "lerna run build --ignore ast-spec",
-    "check:clean-workspace-after-install": "git diff --quiet --exit-code",
-    "check:configs": "lerna run check:configs",
-    "check:docs": "lerna run check:docs",
-    "check:format": "prettier --list-different \"./**/*.{ts,js,json,md}\"",
-    "check:spelling": "cspell --config=.cspell.json \"**/*.{md,ts,js}\"",
+    "build": "npx nx prebuild @typescript-eslint/types && nx run-many --target=build --all --parallel",
```

Comment:
For now, we need to include an explicit reference to the `prebuild` script of `@typescript-eslint/types` because it is unusual in that it has a side-effect on disk outside of `dist/` or `build/` (the common output locations Nx checks for) and it therefore does not play well with the automated caching behavior based on simplistic npm scripts (it would be configurable if we moved to full Nx targets in workspace.json).

Feel free to ping me on slack if you want to cover this in more detail and I can walk you through where this is today vs how it can be configured